src/components/cart/CartSidebar/CartSidebar.tsx

```diff
@@ -57,7 +59,7 @@ function CartSidebar() {
                 classes="cart-sidebar__button"
                 aria-label="Close Cart"
                 icon={<XIcon size={32} />}
-                onClick={() => onDismissTransition()}
+                onClick={() => dismissTransition.current?.()}
```

Comment:
```suggestion
                onClick={dismissTransition.current}
```

Reply:
eslint doesn't allow me to do this.
<img width="979" alt="image" src="https://user-images.githubusercontent.com/1753396/153079746-66f193f6-eaf1-4641-a6d5-1adc5d64f4b8.png">